crossdock/client/ctxpropagation/behavior.go

```diff
@@ -58,6 +59,10 @@ func Run(t crossdock.T) {
 	assert := crossdock.Assert(t)
 	fatals := crossdock.Fatals(t)
 
+	tracer, closer := jaeger.NewTracer("crossdock", jaeger.NewConstSampler(true), jaeger.NewNullReporter())
+	defer closer.Close()
+	opentracing.InitGlobalTracer(tracer)
```

Comment:
This shouldn't be required right? I thought we built our OpenTracing support in a way where you just inject the tracer?

cc @kriskowal @abhinav